app/src/main/java/kth/jjve/xfran/EventPlanActivity.java

```diff
@@ -49,14 +58,15 @@ protected void onCreate(Bundle savedInstanceState) {
         TextView eventDate = findViewById(R.id.eventDate);
         TextView eventStartTime = findViewById(R.id.eventStartTime);
```

Comment:
You are not manipulating these textviews if I am correct. Linking them in the activity is a bit confusing. I would advise taking useless lines away for the sakes of clearness and readability 😄